sources/packages/web/src/components/generic/BCLogo.vue

```diff
@@ -5,8 +5,8 @@
       class="ml-5"
       width="57px"
       height="40px"
-      alt="logo"
       src="../../assets/images/bc_gov_logo_header.svg"
+      alt="StudentAid BC logo"
     />
   </span>
```

Comment:
aria-label?

![image](https://user-images.githubusercontent.com/54600590/186249906-efb576f5-c8b0-4205-94a8-0938947b9257.png)

Reply:
`<v-img` translates alt to aria-label
![image](https://user-images.githubusercontent.com/77353155/186253678-96baa072-431e-47ea-be9b-b56d3716bfbc.png)
https://vuetifyjs.com/en/api/v-img/#props